sklearn/inspection/_plot/partial_dependence.py

```diff
@@ -19,9 +19,13 @@ def plot_partial_dependence(estimator, X, features, feature_names=None,
                             target=None, response_method='auto', n_cols=3,
                             grid_resolution=100, percentiles=(0.05, 0.95),
                             method='auto', n_jobs=None, verbose=0, fig=None,
-                            line_kw=None, contour_kw=None, ax=None):
-    """Partial dependence plots.
+                            line_kw=None, contour_kw=None, ax=None,
+                            individual=False):
```

Comment:
@NicolasHug it might be a good time to add the keyword only syntax and move the individual up in the hierarchy of the keyword (after or before `response_method`)?